src/views/SecurityAndAccess/Sessions/Sessions.vue

```diff
@@ -11,72 +11,80 @@
             {{ $t('pageSessions.alert.message') }}
           </div>
         </alert>
-      </b-col>
-    </b-row>
-    <b-row class="align-items-end">
-      <b-col sm="6" md="5" xl="4">
+      </BCol>
+    </BRow>
+    <BRow class="align-items-end">
+      <BCol sm="6" md="5" xl="4" class="searchStyle">
```

Comment:
SearchStyle is not used. Hence, styling is not proper.
<img width="1674" alt="Screenshot 2024-10-07 at 1 54 45 PM" src="https://github.com/user-attachments/assets/533c18c1-bf63-465a-a329-bfa625bdd394">